src/pages-tab/components/file-picker/file-picker.js

```diff
@@ -32,12 +32,24 @@ export const FilePicker = DefineMap.extend('FilePicker', {
   items: {},
   accept: { type: 'text', default: '' },
 
+  legacyFlattenPath (file) {
```

Comment:
i like this pattern of naming the old ways with 'legacy' - easy way to search this stuff out in the future, or know why it's 'not canjs-i-fied' at a glance